homeassistant/components/shelly/utils.py

```diff
@@ -92,7 +92,4 @@ def get_rest_value_from_path(status, device_class, path: str):
         last_boot = datetime.utcnow() - timedelta(seconds=_attribute_value)
```

Comment:
All this parsing needs to be removed from here. Should happen in device_state_attributes.

Reply:
is already merged ;-)

Reply:
@balloob I started to work on a fix for the drift in time and this will move from here so this function will stay clean and just read data from rest, as it already merged and not related to this commit I suggest to leave it here, I will move it in a different PR, anyhow if needed feel free to move, my PR for this is not ready yet